dictionary/lib/dictionary/word_list.ex

```diff
@@ -1,8 +1,14 @@
 defmodule Dictionary.WordList do
```

Comment:
# -7

I think you missed the mark here. The idea is that the agent contains the wordlist, and runs continuously.

Thus it's initial state should be the list of words.

To get a random word, it does:

~~~ elixir
Agent.get(@me, fn word_list -> Enum.random(word_list) end)
~~~